### PR TITLE
Abductor baton has a cooldown for cuff & sleep mode

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Added a cooldown for the abductor baton's sleep function. Time is 1.5 minutes (up for change)

Added a cooldown for the abductor baton's cuff function. Time is 30 seconds (also up for change)

## Why It's Good For The Game

Abductors are meant to be a stealthy antagonist, one that preys on lone targets and kidnaps them for experimentation. Their kit partially reflects this, with it being comically powerful as an aide - at the expense of not having Rule 4 freedom.

However, this strength, while it does help with the intended strength of abductors, also makes for some extremely unhealthy incentives. An agent can literally just sleep 10 people and keep them cuffed and slept forever for a scientist to process them. This is stupid for all sides, not fun for anyone, and both not intended and not enjoyable emergent gameplay.

To somewhat remedy this, I gave a cooldown to sleeping and cuffing. Stunning is still infinite, though.

You may be concerned about this utterly crippling their ability to take on a single extra person, but there's two items that abductors have that are oft overlooked - the radio silencer and the box of handcuffs. The silencer helps make up for the cd of sleeping, and the handcuffs for restraints.

Let's say there's two scientists in xenobiology. Instead of doing the same thing twice, now you apply the radio silencer, stun-sleep one, stun-cuff the other, cuff the first, and done. 

It encourages abductors to be more tactical as well, making them actually want to wait for there to be only one person around rather than being cocky and confident they can tackle anything with what is probably _the second strongest weapon in the game_ behind the *SM sword*.

In short: This specializes them via mechanical incentives towards the role of predation and abduction rather than being a one-man army demigod somewhat restricted by abductor policy.

Timers are up for change, I have no special attachment to them.

## Changelog

:cl:
balance: Abductor baton has a cooldown for cuff & sleep mode
/:cl:

### CODE REVIEW

**Comment by MrMelbert** [comment]
1.5 minutes do be pretty crazy

**Comment by Jacquerel** [comment]
If you want to remove their ability to control groups then make batoning a fourth person remove the effect from the first or something like that. But it's ultimately pretty intended that if they drop into an area with only a small number of people, they should be able to do what they want.

**Comment by EnterTheJake** [comment]
I think this is unecessary.

Abductors have stupidly strong tools for a couple of reasons.

1) they are a team antag, poor cordination usually leads to the agent death.

2) They don't round remove people, they just open up new avenues of conflict between the crew.

Abductors having tools to fight multiple people is fine, sometimes finding an isolated target is just not an option, we don't want to encourage people to NEVER take risks, it's just boring for all parties involved.

And honestly, if a lone agent is capable of single-handedly incapacitate 10 people and not get his ass, shoved, slipped or riddled with holes that's a massive skill issue on the crew and it's probably deserved.

**Comment by MrMelbert** [comment]
> If you want to remove their ability to control groups then make batoning a fourth person remove the effect from the first or something like that. 

I always planned on one day changing it into a "sleep field", which you can have on max 3 people and other people can attack to break (or you can just wait it out)

**Comment by Striders13** [comment]
abductors are a restricted antag on lrp they don't need nerfs

**Comment by moocowswag** [comment]
Will the cooldown be obvious? This is a rare to get antag and i can only imagine it would be frustrating if an abductor went to go after someone else thinking cooldown was over

**Comment by iwishforducks** [comment]
The idea behind this PR is good. It's annoying when abductors have like 5 people chain slept. It takes people out of the game for far too long. It's hardly about the balance itself IMO, since they still have the ability to stun people and silence their radios. I think the handcuff cooldown is unnecessary, though, or at the very least should only be something like 5 or 10 seconds. The sleep ability should have two charges though IMO. Maybe even lower the cooldown? I'm not sure exactly how long people stay asleep for.

**Comment by Loafin34** [comment]
i thought this was a universal delay on the baton itself, but on the same person? yeah it's fair. maybe like say something as a danger like (You shouldn't overstimulate the patient, it might ruin the testing!)

**Comment by ATH1909** [comment]
I don't see the point of the cuffing cooldown? As you said, the abductors can just use normal cuffs, so it's pointless except for sometimes confusing an agent who doesn't know The Meta.

I don't have a comment on the sleep mode cooldown, but the cuffing cooldown shouldn't be added.

**Comment by DaydreamIQ** [comment]
Cuff Cooldown is irrelevant since abductors spawn with alien cuffs, a minute and a half is kind of ass if you ever need to take down someone who's not alone. Considering this is an antag that isn't designed to be acting as a full antag it seems silly to nerf the thing which helps them achieve their goals so absurdly.

**Comment by Deadgebert** [comment]
I agree that a good abductor is pretty oppressive. I think the sleep cooldown is a poor way to balance it. They should probably be able to take 2-3 people on at a time but no more than that. With melbert's recent stam crit change, you might not even be able to reliably keep 2 extra people stunned long enough to cuff them manually. 
I also want you to think about the down time if there is a failed abduction, a minute and half is a long time to wait per attempt.
If you're specifically looking to stop them from keeping 10 people slept while the scientist processes them, maybe use a charge system of some sort. Maybe like 3 charges that regenerate when you go back to your ship or something. I don't actually know a great solution, but this change is pretty bad.

**Comment by carlarctg** [comment]
3 sleep fields that recharge when you go up ship sounds great
